### PR TITLE
implement non-`connect` based method to check port readiness

### PR DESCRIPTION
**Pull request by dimas3452**

### CODE REVIEW

File: pyrsp/utils.py [hunk 1]
```diff
@@ -16,10 +16,11 @@
 
 # (C) 2014 by Stefan Marsiske, <s@ctrlc.hu>
 
-from socket import SO_REUSEADDR, SOL_SOCKET, socket, AF_INET, SOCK_STREAM
+from socket import socket, AF_INET, SOCK_STREAM
 from time import time
 from json import JSONDecoder, JSONEncoder
 from six import PY3
+from psutil import net_connections
```
Comment (stef) [inline pyrsp/utils.py]: is psutil a new dependency or is psutil packaged in the core py modules?

Comment (dimas3452) [reply pyrsp/utils.py]: `psutil` is  a new dependency.

Comment (stef) [reply pyrsp/utils.py]: ah, then we should add it to setup.py also i guess.

Comment (stef) [reply pyrsp/utils.py]: actually i saw the import. i was just not sure psutil was not already used elsewhere

**Comment by stef** [comment]
<3
as always, great work. thank you very much!

**Pull request merged**